assets/js/components/video.js

```diff
@@ -0,0 +1,12 @@
+export default class {
+	constructor() {
+		document.querySelectorAll(".video-embed > a").forEach((element) => {
+			element.addEventListener("click", this.click);
+		});
+	}
+
+	click(e) {
+		e.preventDefault();
+		e.currentTarget.parentElement.outerHTML = e.currentTarget.dataset.iframe;
```

Comment:
Because we replace the `outerHTML` of the whole container, the overlay is completely removed from DOM.

Reply:
Yes, my suggestion is to additionally remove the text from all other video embeds.

Reply:
I'm confused...

Reply:
Ah, you mean once a user consents to a any video, all other videos will load by default?

Reply:
Not quite, because if we load them all, the buzz overview page has multiple videos autoplaying at once. That's why I thought, we only hide the text overlay for these others, so they look like a normal video with just the play button.

Reply:
I removed `autoplay` together with the `sessionStorage` change for that reason. The `autoplay` attribute doesn't seem to be working reliably anyway as YouTube often ignores it because it cannot reliably detect if user interaction was involved.

I have now tried it as you suggest to only remove the text (see last commit), but the end result is confusing IMO. Without the text, you don't see why you need to click twice if `autoplay` doesn't work. Also it causes flickering when switching pages.

But we have three implementations to choose from now. TBH I'd vote for the first implementation with no `sessionStorage` at all. It works most reliably.

Reply:
I like the newest one, but for me the autoplay also works always (which then made all videos play on the buzz overview with the second implementation). So for me the order would be

- remove all text overlays
-  initial implementation
- remove all overlays

If you don't like the text overlay variant or, rather, the autoplay doesn't reliably work, let's go with the initial one.

Reply:
TBH I can't get any of my browsers (Safari, Chrome, Firefox) to autoplay with the second version. Also I removed the `autoplay` attribute there, so it's even more weird.

Reply:
With the third version I can sometimes get autoplay to work. In Firefox it's quite reliable, in Safari it works sometimes, but in Chrome it never works for me. And without autoplay, I think the removed text is quite confusing.

Reply:
Let's go with the initial version then.